templates/careers/administration.html

```diff
@@ -29,14 +29,11 @@ <h1 class="p-heading--2"><strong>Administration</strong></h1>
   <div class="u-fixed-width">
     <hr class="is-fixed-width u-no-margin--bottom">
     <h2 class="p-text--x-small-capitalised u-align-text--x-small-to-default">Working here</h2>
-  </div>  
+  </div>
   <div class="p-strip is-shallow u-no-padding--top">
     <div class="row">
       <div class="col-6">
-        <h3 class="p-heading--2">Projects you'll work on</h3>
```

Comment:
@lyubomir-popov are you happy with this layout? This is a bit different from the rest of the dept pages as there isn't a header for this section
![Screenshot from 2023-02-09 17-57-29](https://user-images.githubusercontent.com/17607612/217884405-1fd57350-7dbb-4e2b-ac21-a9f5328c6efe.png)

Reply:
Good spot, this is the reason for the Design needed label on this PR :+1:

Reply:
Ah sorry, didn't notice the label

Reply:
May I ask - why is this page diverging from the rest in terms of copy? This will force us to make the design inconsistent by e.g. changing "Working here" to an h2. Is it possible to discuss this with the stakeholder responsible for the copy?

Reply:
If it helps, when I asked Gloria yesterday she said it was because not everyone filled out that extra section for projects

Reply:
Can we fill it in for them? e.g. "Administration at Canonical"

Also, can we work with Gloria to define which parts of the page are optional? Treating headings as optional will make the site inconsistent. Unless someone has specific objections about this, wouldn't it be wiser to work with stakeholders to achieve consistency?

Reply:
btw it says "woking"
![image](https://user-images.githubusercontent.com/2741678/218138308-ec3e30f6-39fb-4a13-9b8a-ec76a398004d.png)

Reply:
I'll give a +1 as visually it is ok, but I think it would be good to initiate a conversation - having 12 pages that follow the same format, but with varying hierarchy makes it difficult to keep them consistent.

Reply:
please fix the typo though :)

Reply:
Fixed 🙂